scripts/templates/ldrddi.cpp.mako

```diff
@@ -233,35 +224,21 @@ ${tbl['export']['name']}(
 
     ${x}_result_t result = ${X}_RESULT_SUCCESS;
 
-    bool atLeastOneplatformValid = false;
     // Load the device-platform DDI tables
     for( auto& platform : ur_loader::context->platforms )
     {
         if(platform.initStatus != ${X}_RESULT_SUCCESS)
             continue;
         auto getTable = reinterpret_cast<${tbl['pfn']}>(
             ur_loader::LibLoader::getFunctionPtr(platform.handle.get(), "${tbl['export']['name']}"));
-        if(!getTable) 
-            continue; 
-        auto getTableResult = getTable( version, &platform.dditable.${n}.${tbl['name']});
-        if(getTableResult == ${X}_RESULT_SUCCESS) 
-            atLeastOneplatformValid = true;
-        %if tbl['experimental'] is False:
-        else
-            platform.initStatus = getTableResult;
-        %endif
+        if(!getTable)
+            continue;
+        platform.initStatus = getTable( version, &platform.dditable.${n}.${tbl['name']});
     }
 
-    %if tbl['experimental'] is False: #//Experimental Tables may not be implemented in platform
-    if(!atLeastOneplatformValid)
-        result = ${X}_RESULT_ERROR_UNINITIALIZED;
-    else
-        result = ${X}_RESULT_SUCCESS;
-    %endif
-
     if( ${X}_RESULT_SUCCESS == result )
     {
-        if( ( ur_loader::context->platforms.size() > 1 ) || ur_loader::context->forceIntercept )
+        if( ur_loader::context->platforms.size() == 0 || ur_loader::context->platforms.size() > 1 || ur_loader::context->forceIntercept )
```

Comment:
`ur_loader::context->platforms.size() >= 0`

Reply:
No, this code is here to make direct calls to the adapter if there's only one. I did `... != 1` instead ;)
Thanks. Done.